src/hooks/useAccounts.tsx

```diff
@@ -187,7 +191,7 @@ const useAccountsContext = () => {
               log('useAccounts/decryptSignature', error);
               dispatch(clearSavedEncryptedSignature());
             }
-          } else if (sourceAccount.encryptedSignature) {
+          } else if (sourceAccount.encryptedSignature && geo && !blockedGeo) {
```

Comment:
i think you don't need the "&& geo"" part bcuz blockedGeo already accounts for it

Reply:
I did some testing, and without "&& geo", the geo can be `undefined` initially which would cause !blockedGeo to be false, which would trigger this autoconnect before geo is fetched. so I added this. 

I figure only down side is that autoconnect would not work if geo is not successfully fetched, but I would argue that we do not want to autoconnect if we couldn't not fetch for geo.

Reply:
Hmmm I wonder if the best solution here is to have isBlockedGeo return true if !geo 🤔 or would that lead to strange UI behavior